blocks/cards/cards.js

```diff
@@ -55,3 +50,66 @@ export default function decorate(block) {
     });
   }
 }
+
+async function fetchOffices(url) {
+  const response = await fetch(url);
+  const json = await response.json();
+  return json.data;
+}
+
+export default async function decorate(block) {
+  const cards = [...block.children];
+
+  // Check for a title row
+  if (!block.classList.contains('shade-icon') && block.children[0].children.length === 1) {
+    const wrapper = cards.shift();
+    const title = wrapper.children[0];
+    title.classList.add('title');
+    wrapper.remove();
+    block.prepend(title);
+  }
+  // Check for JSON data
+  if (cards.length === 1 && cards[0].children.length === 1) {
+    const url = cards.shift();
+    const offices = await fetchOffices(url.querySelector('a[href]').href);
```

Comment:
This seems highly specific for Office data - if this block will only be used for offices, then we should name it "office" or something. so as not to conflate with generic card use case.